geaflow/geaflow-dsl/geaflow-dsl-runtime/src/test/resources/query/jdbc_scan_001.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE kafka_source (
+	user_name varchar,
+	user_count bigint
+) WITH (
```

Comment:
use `jdbc_source` is better? same below

Reply:
Oh, I forgot to change the name when I copy and paste the sql file.